### PR TITLE
[minor] update known_translations

### PR DESCRIPTION
**Pull request by cloud-fan**
## What changes were proposed in this pull request?

update known_translations after running `translate-contributors.py` during 2.4.0 release

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @gatorsmile

**Review by srowen — APPROVED** [review]

**Review by dongjoon-hyun — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Retest this please.

File: dev/create-release/known_translations [hunk 1]
```diff
@@ -203,3 +203,61 @@ shenh062326 - Shen Hong
...
+caneGuy - Kang Zhou
+chaoslawful - Xiaozhe Wang
+cluo512 - Chuan Luo
+codeatri - Neha Patil
+crafty-coder - Carlos Pena
+debugger87 - Chaozhong Yang
+e-dorigatti - Emilio Dorigatti
+eric-maynard - Eric Maynard
+felixalbani - Felix Albani
+fjh100456 - fjh100456
```
Comment (gatorsmile) [inline dev/create-release/known_translations]: Jinhua Fu

File: dev/create-release/known_translations [hunk 2]
```diff
@@ -203,3 +203,61 @@ shenh062326 - Shen Hong
...
+guoxiaolongzte - Xiaolong Guo
+heary-cao - Xuewen Cao
+huangweizhe123 - Weizhe Huang
+ivoson - Tengfei Huang
+jinxing64 - Jin Xing
+liu-zhaokun - Zhaokun Liu
```
Comment (gatorsmile) [inline dev/create-release/known_translations]: Also add
> KaiXinXiaoLei - LeiLei Hu

**Comment by wangyum** [comment]
Update line 132:
```
wangyum - Yuming Wang
```
How about add:
```
lipzhu - Lipeng Zhu
daviddingly - David Ding
laskfla - Keith Sun
```

**Comment by cloud-fan** [comment]
Note that, these updates are generated by the script not me. If someone is not in the list, it means the script can figure out the full name without translation.

File: dev/create-release/known_translations (hunk 1 above)
Comment (cloud-fan) [reply dev/create-release/known_translations]: ah I missed this one.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by gatorsmile** [comment]
Thanks! Merged to master.

**Pull request closed without merging**